wpeverest/user-registration
# Enhance/phone field

### All Submissions:






### Changes proposed in this Pull Request:



[intl-tel-input](https://intl-tel-input.com/) added to enable Smart phone field validation .

### How to test the changes in this Pull Request:

1. Navigate to the form builder and add a phone field in the builder.
2. Choose the smart format for that field.
3. Check for an old field without a smart format. It should be smart now onward.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Introduced Smart format for Phone fields.
